### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [WeblateOrg/weblate#6654] [General] [answered]
Would a Mastodon account for this project be possible?

**Comment by nijel** [comment]
Do you know any social media management tool which supports Mastodon? We would like to be present there, but preferably all our social media would be managed by a single tool (which we are currently looking for, it should at least also support Facebook, Twitter and LinkedIn).

**Reply by linux-lukas** [reply]
There would be the crossposter moa.party that could mirror Instagram and Twitter content to Mastodon and probably the other way around from Mastodon. 

I don't have any experience with this myself - unfortunately.

But maybe it helps you, @nijel

**Reply by nijel** [reply]
Cross posting is the easier part. The harder part is being able to respond there.

**Reply by linux-lukas** [reply]
Then I don't seem to know of any. 

There was once an idea for a COVID-19 info bot to transfer the programmed code into its own library (https://github.com/eknoes/covidbot/issues/127). Perhaps this could be used to build a FLOSS social media management software?

**Reply by linux-lukas** [reply]
I think it would be easy to extend the library with a "Mastodon Reponse & Display" function.

**Reply by linux-lukas** [reply]
I have looked into this a little more. The ticket system Zammad could manage things from Mastodon? @nijel

**Reply by nijel** [reply]
Well, Zammad cannot reasonably well handle Facebook and Twitter (we're using it). Also, I don't see a way to integrate Mastodon in Zammad.

**Comment by KaKi87** [comment, marked as answer]
On [@weblate@fosstodon.org](https://fosstodon.org/@weblate) since late 2022.
